AGENTS.md

```
@@ -0,0 +1,43 @@
+# Developer Docs Agent Context
+
+## Purpose
+Internal developer documentation for Plane EE.
+
+## Structure
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)